README.md

```diff
@@ -23,6 +23,16 @@ OFRAK equips users with:
 
 See [ofrak.com](https://ofrak.com) for more details.
 
+### GUI Frontend
+The GUI view provides a navigate-able resource tree, and for the selected resource: metadata, hex navigation, and an
+entropy / byteclass / magnitude map sidebar. The GUI also allows for actions normally available through the python API
+like commenting, unpacking, analysis, modification and packing of resources.
+
+<div align="center">
+<img id="ofrak-animation" src="docs/assets/ofrak_gui_1.png" style="max-width:1000px;width:100%">
```

Comment:
No need to change anything here, but worth noting that many Markdown viewers (such as the GitHub one) don't respect `style` attributes.